src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md

```diff
@@ -2,26 +2,33 @@
 
 ## Unreleased
 
-* Breaking change: Rename package to OpenTelemetry.Instrumentation.AWSLambda
-  (remove ".Contrib") ([#593](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/593))
-* Breaking change: Move public class AWSLambdaWrapper out of Implementation subnamespace
+## 1.1.0-beta2
+
+Release PR: [#590](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/590)
+
+This is the first release with the new package name `OpenTelemetry.Instrumentation.AWSLambda`.
+
+* BREAKING (API, behavior): Rename package to OpenTelemetry.Instrumentation.AWSLambda
+  (remove ".Contrib") ([#593](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/593)).
+  This also affects the `ActivitySource` name (superseding [#534](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/534)).
+* BREAKING (API): Move public class AWSLambdaWrapper out of Implementation subnamespace
```

Comment:
Let's be consistent and use quotes also here: `AWSLambdaWrapper`, `Implementation `